CustomerOrganizer/CustomerOrganizer/FileIO.py

```diff
@@ -6,12 +6,12 @@
 # Save and load customer's data file.
 class FileIO():
     DATA_FOLDER_NAME = "data"
-#    DATA_FILE_NAME = "test.txt"
     FORMATTED_DATE = date.today().strftime("%m/%d/%y")
+    file_location = ""
     file = None
 
     # saves file, creates new file if it doesnt already exist
-    def save_file(self, file_data, file_name):
+    def save_file(self, file_data, file_name = "John Doe 0000000"):
```

Comment:
Change this to use underscores instead of spaces. (It's just good naming convention to not use spaces in file names.) I'm also wondering if it's better to do last name first. So maybe change this to something like "Doe_John_0000000". We might want to discuss this with Taylor and see what he prefers.

Reply:
"Doe_J_00"

Reply:
fixed